starknet_py/tests/e2e/tests_on_networks/client_test.py

```diff
@@ -371,22 +374,23 @@ async def test_get_block_with_tx_hashes_new_header_fields(client_goerli_testnet)
     "tx_hash, tx_type",
     [
         (
-            0x7B31376D1C4F467242616530901E1B441149F1106EF765F202A50A6F917762B,
+            0x054270D103C875A613E013D1FD555EDCFF2085FECA9D7B4532243A8257FD5CF3,
             DeclareTransactionV3,
         ),
+        # Find DeployAccountTransactionV3 and uncomment it with correct hash
```

Comment:
Do we have our account deployed as v3 transaction @THenry14 @ddoktorski

Reply:
Nope, but we can easily create `DEPLOY_ACCOUNT_V3` transaction by using `Account.deploy_account_v3` static function.